README.md

```diff
@@ -56,11 +56,59 @@ The structure should appear like so:
 ├── documentation
 └── readme.md
 ```
+## New Directory structure (effective July 10th 2017)
 
+Re-organization is being done to support upcoming profiles work and Data Plane swagger specs. The new structure will appear like below
+ 
++---automation
+|   \---resource-manager
+|       \---Microsoft.Automation
+|           \---2015-10-31
+|               \---examples
++---batch
+|   +---data-plane
+|   |   \---Microsoft.Batch
+|   |       +---2015-12-01.2.2
+|   |       +---2016-02-01.3.0
+|   |       +---2016-07-01.3.1
+|   |       +---2017-01-01.4.0
+|   |       |   \---examples
+|   |       \---2017-05-01.5.0
+|   \---resource-manager
+|       \---Microsoft.Batch
+|           +---2015-12-01
+|           +---2017-01-01
+|           |   \---examples
+|           \---2017-05-01
+|               \---examples
++---billing
+|   \---resource-manager
+|       \---Microsoft.Billing
+|           +---2017-02-27-preview
+|           |   \---examples
+|           \---2017-04-24-preview
+|               \---examples
+└── readme.md
+ 
+### Changing the representation of composite swagger metadata
+
+Currently composite swagger metadata is captured in a .json file. The problem with the current representation is that it does not provide a way to track what versions of swagger specs were used in earlier releases. With the upcoming changes, we will be able to keep track of multiple versions of composition. The metadata will be captured in readme.md file under resource-manager folder. There won’t be a need for .json file. New structure in readme.md file looks like below. 
+Tag: 2017-03-30
```

Comment:
Tag name recommendation is changed to package-2017-03 so folks don't confuse this with API version. Could you please fix `Tag` here accordingly